pkg/portallocator/portallocator_test.go

```diff
@@ -570,17 +570,29 @@ func TestPortRangeAllocatorSyncDeleteGameServer(t *testing.T) {
 		return true, nl, nil
 	})
 
-	_, cancel := agtesting.StartInformers(m, pa.gameServerSynced, pa.nodeSynced)
+	ctx, cancel := agtesting.StartInformers(m, pa.gameServerSynced, pa.nodeSynced)
 	defer cancel()
 
+	// wait for caches
+	require.True(t, cache.WaitForCacheSync(
```

Comment:
We don't need this `cache.WaitForCacheSync` since that's also part of `StartInformers` above.